subsys/fs/Kconfig.fatfs

```diff
@@ -129,6 +129,13 @@ config FS_FATFS_MAX_SS
 	range 512 4096
 	default 512
 
+config FS_FATFS_WIN_ALIGNED
```

Comment:
Rename the same way it is now called in https://github.com/zephyrproject-rtos/fatfs/blob/32f6befc3d4dda481f9649663e5122196d2b7c06/include/ffconf.h#L257

Reply:
Hi @de-nordic 

Done force pushed.

Regards
Boon Khai.